mcodingbot/utils/plugin.py

```diff
@@ -0,0 +1,18 @@
+from __future__ import annotations
+
+import typing
+
+import crescent
+
+if typing.TYPE_CHECKING:
+    from mcodingbot.bot import Bot
+
+__all__: typing.Sequence[str] = ("Plugin",)
+
+
+class Plugin(crescent.Plugin):
+    @property
+    def app(self) -> Bot:
+        # NOTE: Worried that not using `if TYPE_CHECKING` for `Bot` will cause
+        # circular imports later down the line.
```

Comment:
Since it's only used as a type, using `if typing.TYPE_CHECKING` makes sense even if there's no risk of circular imports.

Reply:
Do you want me to remove the comment?

Reply:
I don't see any reason to keep it, personally.